scheduler-driver/src/test/java/gov/nasa/jpl/aerie/scheduler/UncontrollableDurationTest.java

```diff
@@ -219,10 +220,10 @@ public void testScheduleExceptionThrowingTask(){
     final var plan = solver.getNextSolution().get();
     //Activity can be started in [0, 2m] but this activity will throw an exception if ran in [0, 1m] so it is scheduled at 2m (as being the second bounds the rootfinding tries before search).
     assertTrue(TestUtility.containsActivity(plan,
-                                            planningHorizon.fromStart("PT120S"),
-                                            planningHorizon.fromStart("PT120S"),
+                                            planningHorizon.fromStart("PT1M38.886061S"),
+                                            planningHorizon.fromStart("PT1M38.886061S"),
                                             problem.getActivityType("LateRiser")));
-    assertEquals(3, problem.getSimulationFacade().countSimulationRestarts());
+    assertEquals(4, problem.getSimulationFacade().countSimulationRestarts());
```

Comment:
Why is there an additional restart now?

Reply:
because before, the second bound (x1) was tested immediately after x0 and was outside the non-working interval. The optimization removes x1.